src/Services/publicApiService.js

```diff
@@ -2,15 +2,30 @@ import axios from 'axios';
 
 const config = {
   headers: {
-    Group: 1, //Aqui va el ID del equipo!!
+    Group: 105, //Aqui va el ID del equipo!!
   },
 };
 
-const Get = () => {
-  axios
-    .get('https://jsonplaceholder.typicode.com/users', config)
-    .then((res) => console.log(res))
-    .catch((err) => console.log(err));
+const publicApiGet = async (route, id) => {
+  let url;
+
+  if (id === undefined || id === null) {
+    url = `http://ongapi.alkemy.org/api${route}`;
```

Comment:
instead of create the route here get the url directly from the parameters to do not depend on the alkemy url